templates/tempest.j2

```diff
@@ -10,8 +10,11 @@ flavor_ref_alt = {{ tempest_second_flavor_uuid }}
 fixed_network_name = {{ tempest_network_name }}
 
 [object-storage]
-swift_operator_role = member
-swift_reseller_admin_role = admin
+swift_operator_role = {{ rally_tempest_swift_operator_role }}
+swift_reseller_admin_role = {{ rally_tempest_swift_reseller_admin_role }}
+
+[object-storage-feature-enabled]
+discoverability = {{ rally_tempest_swift_discoverability }}
```

Comment:
{{ rally_tempest_swift_discoverability }} is not defined in defaults. Please add it there with some example how it can be used.

Reply:
Yes, I will do that and add notes.